pom.xml

```diff
@@ -7,7 +7,7 @@
 	<artifactId>shopify-sdk</artifactId>
 	<name>Shopify SDK</name>
 	<description>Java SDK for Shopify REST API.</description>
-	<version>1.0.0-SNAPSHOT</version>
+	<version>1.0.0.REFUNDS-SNAPSHOT</version>
```

Comment:
So I changed this up to work a little differently then `bigcommerce-sdk`.

If you look at https://github.com/ChannelApe/shopify-sdk/blob/master/.travis.yml you will notice in the deploy section that only master, develop, and tag builds are going to get deployed. Therefore feature and hotfix branches should be updated to their target version.

Please change this back to 1.0.0-SNAPSHOT

Reply:
If this works out, I will update `bigcommerce-sdk` to function the same way